src/libraries/Common/src/System/Net/StreamBuffer.cs

```diff
@@ -277,9 +277,9 @@ public void AbortRead()
                 }
 
                 _readAborted = true;
-                if (_buffer.ActiveLength != 0)
+                if (_buffer.ActiveMemory.Length != 0)
```

Comment:
Nit:  A few lines above in the diff we have a `> 0` check, and here we have a `!= 0` check.  There's no difference, is there?

Reply:
Nope, just inconsistency. Will fix.